pkg/tools/gcscleaner/cleaner.go

```diff
@@ -209,7 +210,7 @@ func (r Cleaner) parseErrors(errorMessages []string) error {
 		return nil
 	}
 	errorMessage := strings.Join(errorMessages, "\n")
-	return fmt.Errorf(errorMessage)
+	return errors.New(errorMessage) //nolint:govet
```

Comment:
Why we need `//nolint:govet`?
```suggestion
	return errors.New(errorMessage)
```

Reply:
because errorMessage is not constant and it is not possible to do it in one line